chain-vote/src/private_voting/challenge_context.rs

```diff
@@ -0,0 +1,52 @@
+//! Challenge context for the Unit Vector Zero Knowledge Proof. The common reference string
+//! is a commitment key, and the statement consists of a public key, and the encryption of each
+//! entry of the vector.
```

Comment:
I would rather make this documentation for ChallengeContext, not for the entire module.